modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/portlet/action/EditCommerceOrderMVCActionCommand.java

```diff
@@ -591,39 +596,33 @@ private void _updatePurchaseOrderNumber(ActionRequest actionRequest)
 	}
 
 	private void _updateRequestedDeliveryDate(ActionRequest actionRequest)
-		throws PortalException {
+		throws ParseException, PortalException {
```

Comment:
I think we can just throws Exception at this point as we don't care the type of it